packages/fte/build.sh

```diff
@@ -27,6 +27,7 @@ termux_extract_src_archive() {
 
 termux_step_host_build() {
 	find "$TERMUX_PKG_SRCDIR" -mindepth 1 -maxdepth 1 -exec cp -a \{\} ./ \;
+	patch -p1 < "$TERMUX_PKG_BUILDER_DIR/strlcpy.patch"
```

Comment:
This patch command is not required explicitly. Add `.beforehostbuild` extension with the patch file and it will be applied automatically before host build.

Reply:
Thank you I did not know about that, and now I applied that correction.